types/zpool.nix

```diff
@@ -37,17 +37,6 @@
       default = [ "defaults" ];
       description = "Options to pass to mount";
     };
-    mountRoot = lib.mkOption {
```

Comment:
@Lassulus where we would add mkRemovedOptionModule if we wanted to add them for submodules?